src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -566,7 +566,7 @@ public void updateFeedbackQuestionNumber(FeedbackQuestionAttributes newQuestion)
         }
         
         adjustQuestionNumbers(oldQuestionNumber, newQuestionNumber, questions);
-        updateFeedbackQuestion(newQuestion);
+        updateFeedbackQuestionWithResponseRateCheck(newQuestion);
```

Comment:
Oops, I updated my old comment. I meant to decrease the visibility of the old method from <code>public</code> to <code>private</code>, so in future developers can't accidentally use that method instead.